browser/extensions/api/brave_rewards_api.cc

```diff
@@ -759,15 +768,15 @@ BraveRewardsGetRecurringTipsFunction::Run() {
 }
 
 void BraveRewardsGetRecurringTipsFunction::OnGetRecurringTips(
-    std::unique_ptr<::brave_rewards::ContentSiteList> list) {
+    ledger::PublisherInfoList list) {
   std::unique_ptr<base::DictionaryValue> result(new base::DictionaryValue());
   auto recurringTips = std::make_unique<base::ListValue>();
 
-  if (!list->empty()) {
-    for (auto const& item : *list) {
+  if (!list.empty()) {
+    for (auto const& item : list) {
```

Comment:
Total nitpick, but `const auto` is a little more idiomatic.

Reply:
fixed